python/paddle/nn/clip.py

```diff
@@ -12,9 +12,1078 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# TODO: define the functions to clip gradient of parameter
-from ..fluid.clip import ClipGradByGlobalNorm  # noqa: F401
-from ..fluid.clip import ClipGradByNorm  # noqa: F401
-from ..fluid.clip import ClipGradByValue  # noqa: F401
+import copy
+import warnings
 
-__all__ = []
```

Comment:
no need to modify? we can leave list of \_\_all\_\_ empty

Reply:
ok, I will revert it.